src/acb_matrix.jl

```diff
@@ -1,4 +1,11 @@
-function AcbMatrix(A::AbstractMatrix{Acb}; prec::Integer = precision(first(A)))
+function AcbMatrix(v::AbstractVector; prec::Integer = precision(first(v)))
```

Comment:
`precision(first(v))` will not always be defined and in general I believe it's only for types from `Arblib.jl` that we want to determine the precision in this way, possibly also `BigFloat`. But in general we should use the global precision.

Reply:
So we should define an internal helper function which falls back to the global precision?

Reply:
That might be a good solution! Something like 
```
@inline _precision(x::ArbTypes) = precision(x)
@inline _precision(_) = DEFAULT_PRECISION[]
```
might be good enough? But then we should try to use this only when necessary and prefer `precision` otherwise I think.